### ISSUE TITLE
Favicon not displayed in Safari

### ISSUE THREAD

**Issue by ermakovich** [withastro/astro.build#1273]
Hello,

Thank you for interesting framework! Just started reading the docs. I noticed a little issue, the site's favicon doesn't seem to work in Safari (I'm on version 17.5). Attaching a screenshot:
<img width="180" alt="Screenshot 2024-09-26 at 20 08 42" src="https://github.com/user-attachments/assets/ad35cfd7-f2f7-4a03-bb38-1b40e5eb0822">

I believe this is because Safari doesn't support SVG favicons.

**Comment by anaxite** [comment]
Having a look, there are a few issues with favicons that could be handled together:

- Safari still doesn't support SVG favicons in 2025.
- iOS devices need a particular icon format for their web shortcuts.
- Android devices need particular icon formats and manifests for their web shortcuts.

I'll have a go at this issue.

**Comment by delucis** [comment]
This is a great guide that might help!

https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs

**Comment by anaxite** [comment]
That link confirms the state of favicons in 2025, which hasn't changed much...

- Safari still wants something that's not an SVG.
- Chrome gets fussy with multiple `<link>` favicon elements.
- Firefox is happy to go with the nicest-looking favicon.

The visibility script is where more of the quirks come out.

**Comment by jacobdalamb** [comment]
Update: newest version of MacOS release Tahoe add SVG support 
https://developer.apple.com/documentation/safari-release-notes/safari-26-release-notes

**Comment by delucis** [comment]
Good point. Wait long enough and browsers eventually fix bugs for you 😁 

Given we haven't fixed this so far and newer browsers all support the current setup, I'd say we can treat this as resolved.
